src/app/clusters/mode-base-server/mode-base-server.h

```diff
@@ -194,6 +203,9 @@ class Delegate
      * CopyCharSpanToMutableCharSpan to copy into the MutableCharSpan.
      * @return Returns a CHIP_NO_ERROR if there was no error and the label was returned successfully.
      * CHIP_ERROR_PROVIDER_LIST_EXHAUSTED if the modeIndex in beyond the list of available labels.
+     *
+     * Note: This is used by the SDK to populate the supported modes attribute. If the contents of this list changes,
```

Comment:
Again, number agreement please, between "contents" and "changes".

Reply:
Fixed.